django/core/management/commands/migrate.py

```diff
@@ -309,3 +324,28 @@ def model_installed(model):
             # Deferred SQL is executed when exiting the editor's context.
             if self.verbosity >= 1:
                 self.stdout.write("    Running deferred SQL...\n")
+
+    def describe_operation(self, operation, backwards):
+        """
+        Return a string, intended for `stdout.write`, that describes
+        a migration operation.
+        """
+        if hasattr(operation, 'code'):
+            action = operation.code.__doc__ if not backwards else operation.reverse_code.__doc__
```

Comment:
It looks like `operation.reverse_code.__doc__` will fail with `AttributeError` if `operation.reverse_code` is `None`.